Cargo.lock

```diff
@@ -246,6 +246,22 @@ dependencies = [
  "libloading",
 ]
 
+[[package]]
+name = "ashpd"
+version = "0.9.0"
+source = "git+https://github.com/bilelmoussaoui/ashpd.git?rev=34c0ab8f83cd16c3190ecc4cc51021daa531d89b#34c0ab8f83cd16c3190ecc4cc51021daa531d89b"
+dependencies = [
+ "enumflags2",
+ "futures-channel",
+ "futures-util",
+ "rand",
+ "serde",
+ "serde_repr",
+ "tokio",
+ "url",
+ "zbus",
+]
+
```

Comment:
Could we maybe `[patch.crates-io]`, so it doesn't get duplicated?
I suppose it's not a big deal, and that would be just one more piece of debris to clean up afterward...

Reply:
Or we could just wait a few days...: https://github.com/bilelmoussaoui/ashpd/pull/234#issuecomment-2357986893 :)

Reply:
A few days passed; let's get this in?

Reply:
I don't object.

Reply:
It should be ready for merge now